code/SerialPortStream.cs

```diff
@@ -695,6 +711,7 @@ private int InternalRead(byte[] buffer, int offset, int count)
 
         private delegate int ReadDelegate(byte[] buffer, int offset, int count);
 
+#if !NETSTANDARD15
```

Comment:
Don't worry - my recompile wasn't right. .NET Standard doesn't have this method. https://docs.microsoft.com/en-us/dotnet/core/api/system.io.stream